src/commons/BrowserUtils.ts

```diff
@@ -143,13 +146,16 @@ export namespace BrowserUtils {
    * @param selector element selector
    */
   export function doubleClick(selector: string): void {
-    Reporter.debug(`Double click an element ${selector}`);
+    Reporter.debug('');
+    Reporter.debug(`Double click an element [${selector}]`);
     isVisible(selector);
     tryBlock(
       () => browser.doubleClick(selector),
 
       `Failed to double click on ${selector}`
     );
+    Reporter.debug('SUCCESS: Double Click.');
+    Reporter.debug('');
```

Comment:
in case you want to add new lines between the logs, it should be part of debug/step/error functions of the reporter